homeassistant/components/loqed/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "LOQED Touch Smart Lock",
   "codeowners": ["@mikewoudenberg"],
   "config_flow": true,
-  "dependencies": ["webhook"],
+  "dependencies": ["cloud", "webhook"],
```

Comment:
This will set up "cloud" as a required dependency, meaning it will set up the cloud integration on systems of people that have explicitly disabled it.

Instead, add `cloud` to the `after_dependencies`.

Reply:
Ah thanks. Done!